sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/Models/SignalRCreateOrUpdateProperties.cs

```diff
@@ -53,9 +49,8 @@ public SignalRCreateOrUpdateProperties()
         /// <param name="upstream">Upstream settings when the Azure SignalR is
         /// in server-less mode.</param>
         /// <param name="networkACLs">Network ACLs</param>
-        public SignalRCreateOrUpdateProperties(string hostNamePrefix = default(string), IList<SignalRFeature> features = default(IList<SignalRFeature>), SignalRCorsSettings cors = default(SignalRCorsSettings), ServerlessUpstreamSettings upstream = default(ServerlessUpstreamSettings), SignalRNetworkACLs networkACLs = default(SignalRNetworkACLs))
```

Comment:
This is a breaking change from the previous, stable version. , we should fix this with a customization that adds back the original method signature.

Reply:
Thanks @markcowl . The `hostNamePrefix` is NEVER used,  client input is always ignored and overriden at service side.  Instead it's causing Swaggger correctness issue. We made some changes to fix it: https://github.com/Azure/azure-rest-api-specs/pull/12101.  I understand it's a breaking change, and we got approval from the review board: https://msazure.visualstudio.com/One/_workitems/edit/9028476. Could you please take a look?

Reply:
This change is covered by the breaking change review on this swagger PR, thanks.